Sources/Spezi/Spezi/Spezi.swift

```diff
@@ -105,7 +105,8 @@ public final class Spezi: Sendable { // swiftlint:disable:this type_body_length
     /// Array of all SwiftUI `ViewModifiers` collected using `_ModifierPropertyWrapper` from the configured ``Module``s.
     ///
     /// Any changes to this property will cause a complete re-render of the SwiftUI view hierarchy. See `SpeziViewModifier`.
-    @MainActor var viewModifiers: [any ViewModifier] {
+    @_spi(Spezi)
+    @MainActor public var viewModifiers: [any ViewModifier] {
```

Comment:
Can we ensure that the documentation here is hidden (e.g. internal) to ensure that this doesn't leak in the API docs? Is there a same thing for other `_@spi` exposed elements?

Applies to all the other ones in the PR diff as well.

Reply:
my expectation is that all SPI definitions are excluded from the documentation (same as they don't appear in the swiftinterface) but i'll take a look

Reply:
this is indeed the case. the `Spezi` class' init is already public and marked as `@_spi(Spezi)`, and it shows up neither in the Xcode-generated docs nor in the swiftpackageindex-hosted ones.

Reply:
Nice, thanks for checking this!